catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.registry.schemabindings;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.AssociationType1;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ExtrinsicObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.InternationalStringType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.LocalizedStringType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ObjectFactory;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.OrganizationType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.PersonType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectListType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ServiceBindingType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ServiceType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType1;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ValueListType;
+
+public final class RegistryPackageUtils {
+
+    public static final net.opengis.gml.v_3_1_1.ObjectFactory GML_FACTORY =
+            new net.opengis.gml.v_3_1_1.ObjectFactory();
+
+    public static final net.opengis.ogc.ObjectFactory OGC_FACTORY =
+            new net.opengis.ogc.ObjectFactory();
+
+    public static final ObjectFactory RIM_FACTORY = new ObjectFactory();
+
+    public static final net.opengis.cat.wrs.v_1_0_2.ObjectFactory WRS_FACTORY =
+            new net.opengis.cat.wrs.v_1_0_2.ObjectFactory();
+
+    private RegistryPackageUtils() {
+    }
+
+    public static List<ServiceBindingType> getBindingTypes(RegistryPackageType registryPackage) {
+        List<ServiceBindingType> bindingTypes = new ArrayList<>();
+        if (registryPackage == null) {
+            return bindingTypes;
+        }
+
+        for (ServiceType service : getServices(registryPackage)) {
```

Comment:
If you do it this way, you probably won't need to create the `bindingTypes` `ArrayList` and instead should just be able to return `Collections.emptyList()` in the `if` statement.